### ISSUE TITLE
Another Preview issue with 6.070 wxchordpro

### ISSUE THREAD

**Issue by JoeHartley3** [ChordPro/chordpro#501]
When trying to preview a file, nothing is displayed, and the terminal it's running from shows many lines of "Failed to get GBM device".

This is when both "Ignore default config files" and Presets are un-checked.  When they are checked, I get the "Error: Can't call method "split_fc_aliases" on unblessed reference at /usr/share/perl5/site_perl/ChordPro/Config.pm line 363." error as reported by johnkissel.

The messages pane reports this:
09:29:03 AM: Using styled text editor
09:29:03 AM: Using embedded PDF viewer
09:29:03 AM: Loaded: Walk Right Back - Everly Bros. (52 lines)
09:29:16 AM: Warning: Config error: unknown item diagrams.auto
09:29:16 AM: Warning: Config error: unknown item diagrams.auto
09:29:18 AM: Preview file:///tmp/6Ap4IuZvwy.pdf
09:29:18 AM: 2 messages
09:29:18 AM: Warning: Problems found.

The preview file is created and looks fine using an external PDF viewer (atril, in this case). 
The error lines "Config error: unknown item diagrams.auto" also appear if I run the CLI version but don't appear to affect the output.

## System information
 - OS: Arch Linux - kernel 6.12.7-arch1-1
 - ChordPro installed via CPAN, wxWigets and QuickJS installed via AUR

$ chordpro --about
   [ snip ]
Run-time information:
  ChordPro core          6.070     
  Perl                   v5.40.0    (/usr/bin/perl)
  Resource path          ~/.config/chordpro
                         /usr/share/perl5/site_perl/ChordPro/res

Modules and libraries:
  Storable               3.32      
  Object::Pad            0.813     
  Text::Layout           0.038     
  HarfBuzz::Shaper       0.027     
    library              10.1.0    
  File::LoadLines        1.046     
  PDF::API2              2.047     
  SVGPDF                 0.090     
  Font::TTF              1.06      
  JavaScript::QuickJS    0.21      
  JSON::Relaxed          0.097     
  JSON::XS               4.03

**Comment by sciurius** [comment]
"Failed to get GBM device" seems to be Wayland related, I'm afraid that is out of our league.

You checked "Ignore default config files" and unchecked "Presets'. Do you have a custom config file? I'm wondering where the `diagrams: auto` comes from.

"Error: Can't call method "split_fc_aliases" indicates an error in one of the config files.

**Comment by JoeHartley3** [comment]
There's no Wayland involved here, I'm X11 all the way.  My window manager is Fluxbox.  It's a pretty stripped down environment.

I don't recall ever customizing the config file before yesterday, but I've been using this program since the days of Martin Leclerc so it's possible I did at one point.  Unfortunately yesterday's edit means I've lost any previous timestamps.

The file size compared to the default is quite different:
```
$ ls -l chordpro.json /usr/share/perl5/site_perl/ChordPro/res/config/chordpro.json 
-rw-r--r-- 1 jh   jh   23725 Dec 30 17:44 chordpro.json
-r--r--r-- 1 root root 28012 Dec 20 06:13 /usr/share/perl5/site_perl/ChordPro/res/config/chordpro.json
```

Following something I saw after googling about a similar GBM error, I found that I could get the preview running by using this command:

`
$ WEBKIT_DISABLE_COMPOSITING_MODE=1 wxchordpro
`
This seems to be related to my use of Nvidia drivers.

Closing this bug as it does not seem to be a problem with ChordPro.
